nimbus.fml.yaml

```diff
@@ -508,6 +508,22 @@ features:
             "move-actions": true,
             "toolbar-changes": true
         }
+  creditcard-autofill:
```

Comment:
I would call this `credit-card-autofill` as credit card is, technically, two words.